waku/v2/protocol/waku_message.nim

```diff
@@ -31,12 +31,16 @@ type
     # this field will be used in the rln-relay protocol
     # XXX Experimental, this is part of https://rfc.vac.dev/spec/17/ spec and not yet part of WakuMessage spec
     proof*: RateLimitProof
+    # The ephemeral field indicates if the message should
+    # be stored. bools and uints are 
+    # equivalent in serialization of the protobuf
+    ephemeral*: bool
     
    
 
 # Encoding and decoding -------------------------------------------------------
 proc init*(T: type WakuMessage, buffer: seq[byte]): ProtoResult[T] =
-  var msg = WakuMessage()
+  var msg = WakuMessage(ephemeral: false)
```

Comment:
Is initialising the field here necessary? Empty initialisation should initialise all fields with a default value, including `false` for booleans.

Reply:
It isn't necessary, just added it in to be explicit